--- a/drivers/hid/Makefile 2022-12-17 20:09:21.691387941 +0100
+++ b/drivers/hid/Makefile 2022-12-17 20:10:23.993185886 +0100
@@ -151,3 +151,5 @@
 obj-$(CONFIG_AMD_SFH_HID)       += amd-sfh-hid/

 obj-$(CONFIG_SURFACE_HID_CORE)  += surface-hid/
+
+ obj-y += ipts/
--- a/drivers/hid/Kconfig 2022-12-17 20:09:15.714409690 +0100
+++ b/drivers/hid/Kconfig	2022-12-17 20:10:08.448232230 +0100
@@ -1254,4 +1254,6 @@

 source "drivers/hid/surface-hid/Kconfig"

+source "drivers/hid/ipts/Kconfig"
+
 endmenu
